src/components/ProposalDetailPanel.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { UsersIcon } from '@heroicons/react/24/solid';
+import {
+  Panel,
+  PanelBody,
+  PanelHeader,
+  PanelTag,
+  PanelTitle,
+  PanelTitleTags,
+  PanelTitleWrapper,
+} from './Panel';
+import { ProposalTable } from './ProposalTable';
+
+interface ProposalDetailPanelProps {
+  title: string;
+  author: string;
+  authorId: string | undefined;
+  version: number;
+  values: {
+    fieldName: string,
+    position: number,
+    value: string,
+  }[],
+}
+
+const ProposalDetailPanel = ({
+  title,
+  author,
+  authorId,
+  version,
+  values,
+}: ProposalDetailPanelProps) => (
+  <Panel>
+    <PanelHeader>
+      <PanelTitleWrapper>
+        <PanelTitle>
+          {title}
+        </PanelTitle>
+        <PanelTitleTags>
+          <PanelTag>
+            <UsersIcon className="icon" />
+            {author}
+            {authorId && ` (${authorId})`}
+          </PanelTag>
+        </PanelTitleTags>
```

Comment:
After talking with @jim-mcgowan, his main feedback was that we should be highlighting the applicant above the proposal title. The EIN is of secondary importance, and should not be given the same weight as the promoted organization name; he suggested displaying it below, but maybe alongside in gray might work too - I'll play with it a bit, or possibly wait for @reefdog's feedback.

Reply:
Here's what I have right now:

![Screen Shot 2023-03-31 at 18 54 49](https://user-images.githubusercontent.com/1494855/229260308-3fa009ed-0046-4653-a4e0-bb51e2154129.png)